code/create_app.py

```diff
@@ -427,6 +427,7 @@ def speech_config():
             if response.status_code == 200:
                 return {
                     "token": response.text,
+                    "key": speech_key,
```

Comment:
This and all of the other changes in this PR, except `03_Delete_Data.py` look unrelated to the PR title and description. Are they supposed to be here?

Reply:
reverted changes